guides/source/debugging_rails_applications.md

```diff
@@ -461,11 +490,11 @@ the `thread` command (or the abbreviated `th`). This command has a handful of
 options:
 
 * `thread`: shows the current thread.
-* `thread list`: is used to list all threads and their statuses. The plus +
-character and the number indicates the current thread of execution.
-* `thread stop _n_`: stop thread _n_.
-* `thread resume _n_`: resumes thread _n_.
-* `thread switch _n_`: switches the current thread context to _n_.
+* `thread list`: is used to list all threads and their statuses. The current
+thread is marked with a plus (+) sign.
+* `thread stop n`: stops thread _n_.
+* `thread resume n`: resumes thread _n_.
+* `thread switch n`: switches the current thread context to _n_.
```

Comment:
> That's why I wrapped two places to 80 chars, because this whole section of the guide (except for those 2 places) is wrapped to 80 chars.

Agree, but this is not consistent across this guide or all others. We don't follow that.

> I think those are exactly the changes I'm including in this PR

Yup, asking you for a lot of trouble, but we will need to change everywhere else 😄 .Thanks!